### PR TITLE
Adds dataset cards to Methodology page

### LINKED ISSUE #144: As an EVCM, I want to see and understand any data sources used in the generation of the score and list, so that I can understand the data & methodology (FRONTEND ONLY).
**Description**
The need for both EVCM and agency partners is to know the provenance of the datasets and information about the scoring methodology used to calculate the cumulative index score and determine which communities are prioritized. 

**Solution**
This will require a separate Data & methodology page with a section that describes the scoring methodology and a section that shows individual cards for each dataset that is used in the latest version of the cumulative index. 

Note that this is just about getting the dataset information from the JSON and displaying it, not other design updates.

**Describe alternatives you've considered**

**Links to user research or other resources**

**Tasks**
- [ ] Data set info should draw on the JSON file that is generated from YAML files for datasets published in the repo
 - [ ] Ensure cards match format shown in latest sprint design which includes link to data source

**Not in scope**
 - Cards don't need to be sorted or moveable

**Definition of "Done"**
Site with date representing latest set of datasets and scoring methodology planned for 7/27 launch can be navigated on http://usds-geoplatform-justice40-website.s3-website-us-east-1.amazonaws.com/justice40-tool/f12ab4/en/cejst/ (or similar)

### PR DESCRIPTION
**Pull request by vim-usds**
closes #144 
Recreating this PR to troubleshoot build failure

### CODE REVIEW

File: client/src/components/DatasetCard/index.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+import * as styles from './datasetCard.module.scss';
+
+interface IDatasetCardProps {
+  key: number,
+  datasetCardProps: { [key:string]: string }
+}
+
+const DatasetCard = ({key, datasetCardProps}:IDatasetCardProps) => {
+  const intl = useIntl();
+  const messages = defineMessages({
```
Comment (NatHillardUSDS) [inline client/src/components/DatasetCard/index.tsx:13]: Nice!

**Review by NatHillardUSDS — DISMISSED** [review]
Awesome! Looks great.

**Review by NatHillardUSDS — DISMISSED** [review]

File: client/src/components/DatasetCard/datasetCard.module.scss [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+}
+
+.datasetCardIndicator {
+  margin-top: 0px;
+  font-size: x-large;
+  margin-bottom: 0.8rem;
+}
+
+.datasetCardWhatIsIt {
+  margin-bottom: 0px;
```
Comment (TomNUSDS) [inline client/src/components/DatasetCard/datasetCard.module.scss:15]: Think it's just `0` (the `px`) is not needed.

File: client/src/components/DatasetCard/datasetCard.module.scss [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  font-size: large;
+  font-weight: bolder;
+}
+
+.datasetCardList {
+  list-style-type: none;
+  padding-left: 0px;
```
Comment (TomNUSDS) [inline client/src/components/DatasetCard/datasetCard.module.scss:22]: `0` vs `0px`

File: client/src/components/J40Alert/j40Alert.module.scss [hunk 4]
```diff
@@ -0,0 +1,5 @@
+.j40Alert {
+  font-size: large;
+  font-weight: 600;
+  padding: 1rem 0rem;
```
Comment (TomNUSDS) [inline client/src/components/J40Alert/j40Alert.module.scss:4]: `0rem` vs just `0`

File: client/src/components/layout.tsx [hunk 5]
```diff
@@ -31,6 +34,7 @@ const Layout = ({children, location}: ILayoutProps) => {
           {conditionalAside}
         </Grid>
       </GridContainer>
+      {isMethodologyPage ? <DatasetContainer />: null}
```
Comment (TomNUSDS) [inline client/src/components/layout.tsx:37]: It is `null` or `<></>` or `< />` ? @NatHillardUSDS (is this a style thing?)

**Review by TomNUSDS — APPROVED** [review]
LGTM. Minor style things that are optional.

File: client/src/components/layout.tsx (hunk 5 above)
Comment (vim-usds) [reply client/src/components/layout.tsx:37]: Good call out! 

TL:DR
It seems the first link says, `< />` creates a React Fragment which might be more of a lift than null. Then it said the TS didn't handle JSX elements returning null. Then the 2nd link shows how the official TS issue from MS allows for null now.

1. https://www.reddit.com/r/reactjs/comments/gtyr61/null_vs_empty_fragment/  , which lead to this:
2. https://github.com/microsoft/TypeScript/issues/11566

Maybe we should make null the standard on conditional renders that return nothing?

**Comment by vim-usds** [comment]
@TomNUSDS , thanks! Just removed the units on all css values that equal 0

File: client/src/components/layout.tsx (hunk 5 above)
Comment (TomNUSDS) [reply client/src/components/layout.tsx:37]: Thanks for doing the research. I'm happy with using `null` would be nice if linters could be configured to double-check. But probably not worth the time investment in the near term.

**Review by TomNUSDS — DISMISSED** [review]
LGTM!

**Review by TomNUSDS — APPROVED** [review]
Sorry for the merge conflict. LGTM.

**Pull request merged**